src/schema/v2/me/myCollectionCreateArtworkMutation.ts

```diff
@@ -115,7 +116,7 @@ export const myCollectionCreateArtworkMutation = mutationWithClientMutationId<
       type: GraphQLString,
     },
     pricePaidCents: {
```

Comment:
There's a costMinor field further up in this file that is still represented as a GraphQLInt. Should that be updated as well?